modules/custom-email/admin/custom-email-admin.php

```diff
@@ -595,6 +644,7 @@ public function user_denial_meta_box() {
 	public function save_settings( $settings ) {
 		$settings['new_user']['admin_disable']   = isset( $settings['new_user']['admin_disable']   );
 		$settings['reset_pass']['admin_disable'] = isset( $settings['reset_pass']['admin_disable'] );
+                $settings['pass_change']['admin_disable'] = isset( $settings['pass_change']['admin_disable'] );

```

Comment:
There is no admin_disable checkbox in the meta box, right? I think this line can be left out.